docs/_docs/extras.md

```diff
@@ -6,6 +6,14 @@ permalink: /docs/extras/
 There are a number of (optional) extra features that Jekyll supports that you
 may want to install, depending on how you plan to use Jekyll.
 
+## Web Highlights and Commenting 
+
+Register your site with [txtpen](https://txtpen.com). Then append 
+
+    <script src="https://txtpen.com/embed.js?site=<your site name>" />
```

Comment:
could you use fenced code blocks? you can find an example in the `Math Support` section just below this one